LocalPlanner: 
  verbose: true
  odom_topic: "/odom"
  goal_lin_tolerance: 0.05
  goal_ang_tolerance: 0.05
  max_safe_lin_velocity: 0.7
  max_safe_ang_velocity: 0.9
  max_safe_lin_displacement: 0.5
  max_safe_ang_displacement: 1.0
  safety_observers: ["arm_skin_observer"]

  MotionPlanner:
    max_linear_velocity: 0.8
    max_angular_velocity: 1.0
    linear_smoother: 0.75
    angular_smoother: 0.75
    time_scaler: 1.25
    waypoints_heading_lookahead: 15
    lookahead: 0.0

  ControllerPID:
    kP_lin: 3.0
    kP_ang: 3.0
    kI_lin: 0.0001
    kI_ang: 0.0001
    kD_lin: 0.0001
    kD_ang: 0.0001
  
  scan_safety_observer:
    verbose: false
    scan_topic: "/scan"
    max_safety_rangevel: 0.5
    unsafe_range: 0.75
    ab_filter: {alpha: 0.1, beta: 0.005}
    enabled: false

  arm_skin_observer:
    verbose: false
    topic_name: "/arm_skin"
    enabled: false